roles/lib_openshift/library/oc_adm_registry.py

```diff
@@ -2049,6 +2056,7 @@ class Service(Yedit):
     cluster_ip = "spec.clusterIP"
     selector_path = 'spec.selector'
     kind = 'Service'
+    external_ips = "spec.externalIPs"
```

Comment:
Not related to this PR directly, but at some point these attributes should probably be renamed to follow a pattern such as ``cluster_ip_key``, etc..